go.mod

```diff
@@ -1,20 +0,0 @@
-module github.com/moov-io/ach
```

Comment:
@bkmoovio Sorry, I just meant to delete your changes from these `go.*` files. We need them to Go modules.